### ISSUE TITLE
I have too many friends

### ISSUE THREAD

**Issue by mcao** [GriffinG1/Custom-Cogs#6]
```
Ignoring exception in command serverfriends
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/discord/ext/commands/core.py", line 62, in wrapped
    ret = yield from coro(*args, **kwargs)
  File "/home/asianboi/Discord-Selfbot/custom_cogs/serverfriends.py", line 25, in serverfriends
    await ctx.send(embed=embed)
  File "/usr/local/lib/python3.6/site-packages/discord/abc.py", line 764, in send
    data = yield from state.http.send_message(channel.id, content, tts=tts, embed=embed, nonce=nonce)
  File "/usr/local/lib/python3.6/site-packages/discord/http.py", line 212, in request
    raise HTTPException(r, data)
discord.errors.HTTPException: BAD REQUEST (status code: 400): Invalid Form Body
In embed: Embed size exceeds maximum size of 2000

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/discord/ext/commands/bot.py", line 886, in invoke
    yield from ctx.command.invoke(ctx)
  File "/usr/local/lib/python3.6/site-packages/discord/ext/commands/core.py", line 470, in invoke
    yield from injected(*ctx.args, **ctx.kwargs)
  File "/usr/local/lib/python3.6/site-packages/discord/ext/commands/core.py", line 71, in wrapped
    raise CommandInvokeError(e) from e
discord.ext.commands.errors.CommandInvokeError: Command raised an exception: HTTPException: BAD REQUEST (status code: 400): Invalid Form Body
In embed: Embed size exceeds maximum size of 2000
```

TL;DR account for more friends in the embed than the embed size allows

EDIT:

```
Ignoring exception in command nickscan
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/discord/ext/commands/core.py", line 62, in wrapped
    ret = yield from coro(*args, **kwargs)
  File "/home/asianboi/Discord-Selfbot/custom_cogs/nickscan.py", line 24, in nickscan
    await ctx.send(embed=embed)
  File "/usr/local/lib/python3.6/site-packages/discord/abc.py", line 764, in send
    data = yield from state.http.send_message(channel.id, content, tts=tts, embed=embed, nonce=nonce)
  File "/usr/local/lib/python3.6/site-packages/discord/http.py", line 212, in request
    raise HTTPException(r, data)
discord.errors.HTTPException: BAD REQUEST (status code: 400): Invalid Form Body
In embed: Embed size exceeds maximum size of 2000

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/discord/ext/commands/bot.py", line 886, in invoke
    yield from ctx.command.invoke(ctx)
  File "/usr/local/lib/python3.6/site-packages/discord/ext/commands/core.py", line 470, in invoke
    yield from injected(*ctx.args, **ctx.kwargs)
  File "/usr/local/lib/python3.6/site-packages/discord/ext/commands/core.py", line 71, in wrapped
    raise CommandInvokeError(e) from e
discord.ext.commands.errors.CommandInvokeError: Command raised an exception: HTTPException: BAD REQUEST (status code: 400): Invalid Form Body
In embed: Embed size exceeds maximum size of 2000
```
Same thing happens with NickScan

**Comment by GriffinG1** [comment]
Delete all your nicks and friends /s

Seriously though, I'll fix it later

**Comment by GriffinG1** [comment]
Hotfixed in https://github.com/GriffinG1/Custom-Cogs/commit/fbf8949522d9bf165c582a62bb5cfe0528408bd5, https://github.com/GriffinG1/Custom-Cogs/commit/052e0782732ebff321a89efa55c448c0c6e43306, https://github.com/GriffinG1/Custom-Cogs/commit/c787c35c2b6acb749656d0623e472baf08930e19, and https://github.com/GriffinG1/Custom-Cogs/commit/84269b7f34ec153b223160c255c91f5b2da3c0ba. Full fix will be done at a later date
